src/OpenTelemetry.Instrumentation.StackExchangeRedis/Implementation/RedisMetrics.cs

```diff
@@ -0,0 +1,59 @@
+// Copyright The OpenTelemetry Authors
+// SPDX-License-Identifier: Apache-2.0
+
+using System.Diagnostics.Metrics;
+using System.Reflection;
+using OpenTelemetry.Internal;
+using OpenTelemetry.Trace;
+
+namespace OpenTelemetry.Instrumentation.StackExchangeRedis.Implementation;
+
+internal class RedisMetrics : IDisposable
+{
+    internal const string OperationMetricName = "db.client.operation.duration";
+    internal const string QueueMetricName = "db.client.queue.duration";
```

Comment:
Are these extra metrics, not yet part of https://github.com/open-telemetry/semantic-conventions/blob/main/docs/database/database-metrics.md?

Reply:
Correct. Its best to start adding those metrics part of the conventions. Once sem. conventions add more, we can come back and add more here too.

Reply:
I removed the queue and network metrics. It will only have `db.client.operation.duration`.